app/views/admin/pages/index.html.erb

```diff
@@ -8,6 +8,7 @@
 <div class="crayons-card p-6 pages__table">
   <% @pages.each do |page| %>
     <div class="flex py-2 items-center">
+      <%= inline_svg_tag("lock.svg", class: "crayons-icon", aria: true, title: "Lock icon") if page.landing_page? %>
```

Comment:
I don't think "Lock icon" communicates what its purpose is to the screen readers. What about "Current landing page" ?

Reply:
Great suggestion!